index.php

```diff
@@ -1,140 +1,91 @@
 <?php
-    session_start();
-    if(session_is_registered(username)) {
-        header("location:admin.php");
-    }
+    require_once 'core/init.php';      
+    if(!User::isLoggedIn())       
+    {     
+        Redirect::to('login.php');        
+    }            
 ?>
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
+<!DOCTYPE html>
+<html>
 <head>
-<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<title>Student Information System</title>
-<link rel="stylesheet" type="text/css" href="css/reset.css" />
-<link rel="stylesheet" type="text/css" href="css/960.css" />
-<link rel="stylesheet" type="text/css" href="css/style.css" />
-<link rel="stylesheet" type="text/css" href="css/jquery-ui-1.8.custom.css" />
-
-<script type="text/javascript" src="ajax/jquery-1.4.1.min.js"></script>
-<script type="text/javascript" src="ajax/jquery-ui-1.8.custom.min.js"></script>
-<script type="text/javascript" src="ajax/main.js"></script>
+    <title>Home</title>
+    <link rel="stylesheet" type="text/css" href="/css/bootstrap.css">
+    <link rel="stylesheet" type="text/css" href="/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="/css/homepagecustom.css">
```

Comment:
URLs shouldn't be absolute, we should remove the leading '/' from all the hrefs, e.g. - 

`href="/css/homepagecustom.css"`

should be

`href="css/homepagecustom.css`